docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,171 @@
+---
+title: ".NET Docs What's New (November 2019)"
+description: "What's new in .NET docs in November 2019."
+ms.date: 2019/12/13
+---
+
+# .NET Docs What's New (November 2019)
+
+Welcome to what's new in .NET docs for November 2019. Listed below are some of the recent docs we've published.
```

Comment:
The title implies that this is for docs published in November, but the list seems too long for that. Also this ("November") seems to conflict with the contributor list which says it's "since our last release notes".  Suggestion is to clarify the period covered, and make the period consistent for docs and contributor list.

Also, it would also be ideal to call out differently docs that are new as opposed to just revised. To be even more useful, revised docs would have a short blurb summarizing what was done to them.

Reply:
@MaximRouiller How did you tell the difference between new and updated docs? I could not find this info in the API values.

Reply:
I had to use the REST API, instead of the GraphQL API.

Reply:
New pages were using ⭐️ emojis as a way to differentiate from purely modified pages. 

In my code, new pages were generated by a single command line, the modified were from a different command line. I haven't used the API so I would need to investigate that issue.

Reply:
I love the :star: emoji. I didn't use it to account for accessibility concerns. In a future PR, I would add it, if we can solve that with alt-text.